### PR TITLE
Fixed a regression where unnecessary whispace was added to rendered html

### PR DESCRIPTION
**Pull request by vxsx**
also added a warning to tests so it's not as easily changed

### CODE REVIEW

**Comment by Brankub** [comment]
👍

File: djangocms_link/templates/cms/plugins/link.html [hunk 1]
```diff
@@ -1 +1 @@
-{% load cms_tags %}{% if object.child_plugin_instances %}<span class="plugin_link">{% endif %}<a href="{{ link }}"{% if target %} target="{{ target }}"{% endif %} {{ object.attributes_str }}>{% for plugin in object.child_plugin_instances %}{% render_plugin plugin %}{% empty %}{{ name }}{% endfor %}</a>{% if object.child_plugin_instances %}</span>{% endif %}
+{% load cms_tags %}{% if object.child_plugin_instances %}<span class="plugin_link">{% endif %}<a href="{{ link }}"{% if target %} target="{{ target }}"{% endif %} {{ object.attributes_str }}>{% for plugin in object.child_plugin_instances %}{% render_plugin plugin %}{% empty %}{{ name }}{% endfor %}</a>{% if object.child_plugin_instances %}</span>{% endif %}
```
Comment (lory87) [inline djangocms_link/templates/cms/plugins/link.html]: no new line at the end of the file

**Comment by lory87** [comment]
otherwise LGTM

File: djangocms_link/templates/cms/plugins/link.html (hunk 1 above)
Comment (vxsx) [inline djangocms_link/templates/cms/plugins/link.html]: that's the whole point of this pull request

**Pull request merged**